modules/mobile/spec/support/fixtures/VAOS_v2_appointments.json

```diff
@@ -827,123 +827,7 @@
     }
   },
   {
-    "id": "145078",
```

Comment:
This data is no longer necessary after the recent change to parse acheron data in the service.